src/scripts/filtering.js

```diff
@@ -25,6 +40,14 @@ const filterPosts = async posts => {
     filterTags && (tags = [...model.tags, ...model.shareTree.flatMap(share => share.tags)].map(str => str.trim().toLowerCase()).join(' '));
     filterCws && (cws = [...model.cws, ...model.shareTree.flatMap(share => share.cws)].map(str => str.trim().toLowerCase()).join(' '));
 
+    if (doHideShares) {
+      const poster = model.postingProject.handle
+      const sharedFrom = model?.shareTree[model?.shareTree.length - 1]?.postingProject?.handle
+      if (!!sharedFrom && poster !== sharedFrom) {
+        $(post).parent().parent().addClass(isShareClass)
```

Comment:
in essence of the first comment, setting `customAttribute` to `hidden` would accomplish the same thing

Reply:
then when the user unchecks the "hide shares" box, we'd have to somehow find all the shares and mutated the DOM to show them again